python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -324,17 +324,18 @@ class RandomForestClassifier(Base):
         cdef size_t params_t64
         if self.n_cols:
             # only if model has been fit previously
-            model_pbuf_bytes = self._get_protobuf_bytes()
+            self._get_protobuf_bytes()  # Ensure we have this cached
             params_t = <uintptr_t> self.rf_forest
-            rf_forest = \
-                <RandomForestMetaData[float, int]*>params_t
+            if params_t:
```

Comment:
we are using a cdef variable value for the check here, i think it would be nice to check if `self.rf_forest` is set instead

Reply:
updated... let me know if this is what you had in mind.